README.md

```diff
@@ -33,40 +31,55 @@ Available modules
 * `status`
 * `task`
 
-Installation
----
+## Installation
 
-After building the collection artifact with `make dist`, you can install the resulting `tar.gz` file with `ansible-galaxy` in the usual ways.
+### Install from Ansible Galaxy
 
-Documentation
----
+The collection is available from Ansible Galaxy, so you can install it via
 
-You can find the inline documentation of each module with `ansible_doc pulp.squeezer.pulp_<...>`.
+    $ ansible-galaxy collection install pulp.squeezer
 
-Testing
----
+### Build locally
+
+Alternatively you can building the collection artifact with
+
+    $ make dist
+
+and install the resulting `tar.gz` file with
+
+    $ ansible-galaxy collection install pulp-squeezer-<version>.tar.gz
+
+## Documentation
+
+You can find the inline documentation of each module with `ansible-doc pulp.squeezer.<module_name>`.
+
+## Testing
 
 Testing is done by running handcrafted playbooks from `tests/playbooks` while playing back prerecorded server answers.
 Using python virtual environments is recommended.
 
-Those playbooks are meant to test one specific module, and are usually structured in three sections:
+There is usually one playbook per module that it is meant to test, but that is not a hard requirement.
+
+The playbooks are usually organized in a three-act structure:
 
- * In the first part, ran against `localhost`, fixtures can be set up.
- * The second part should contain the actual tests and is executed on the virtual host `tests` to allow recording.
- This usually involves calling the module in question several times with varying parameters and verifying it's output.
- * The third part, again for `localhost`, provides the the opportunity to undo part one.
+ * In the first act, the setup, ran against `localhost`, fixtures can be prepared.
```

Comment:
Hey @mdellweg 
I am going to ask a lot of questions. We can have a quick phone conversation to review if it is easier. 

this might be a really stupid question, but does this sentence mean the same thing: 

"In the first act, the setup runs against `localhost` to prepare the fixtures"? 

Also, does "fixtures" mean "a collection of raw fixture data, scripts for compiling that data into a useful format, and make targets for controlling the whole mess." The technical writer in me would suggest we link to where 'fixtures' is defined for the hopeless among us who are reading this for the first time.

Reply:
I was a little carried away by: https://en.wikipedia.org/wiki/Three-act_structure Maybe not the best idea.
The part you started to discuss here is the section someone hopefully reads, once he has decided to get his hands dirty and wants to contribute more modules/features. So i am assuming a some knowledge of programming and ansible here.
I am trying to explain the structure and the mechanics of the tests. In the context of tests, i think fixtures is a _thing_: Some entries you put in the database so your tests have something to chew on.

Reply:
My 2c on this: 

As a Classics graduate, I am delighted to see the narrative models of Aelius Donatus alive and well in the 21st century. My only concern would be that this introduces terminology that is inconsistent with the other Pulp documentation. The "acts" "confrontation", while accurate, requires the user to understand this dramatic framework, which is not necessary and might confuse them. When it comes to usability, writing documentation in a consistent manner across all parts of the product reduces the mental load for the user. They grow to anticipate the way the information is framed.

Reply:
I think the official terminology used in the context of ansible goes like "A playbook contains a list of plays". I'll change it to that.
And if only it helped to get some good discussions started on those lines, it was totally worth it.